spec/PhpSpec/Locator/PSR0/PSR0LocatorSpec.php

```diff
@@ -5,7 +5,7 @@
 use PhpSpec\ObjectBehavior;
 use PhpSpec\Util\Filesystem;
 
-use SplFileInfo;
+use Symfony\Component\Finder\SplFileInfo;
```

Comment:
why this change ?

Reply:
In PHP 8.1, the following error started to make the test suite fail, which I think is related to https://github.com/phpspec/prophecy/issues/414:

```
PhpSpec/Locator/PSR0/PSR0Locator
 137  - it finds all resources from tracked specPath
      8192: SplFileInfo::__construct(): Passing null to parameter #1 ($filename) of type string is deprecated in
      /phpspec/vendor/phpspec/prophecy/src/Prophecy/Doubler/Generator/ClassCreator.php(49) : eval()'d code line 6
```

Given that [Filesystem](https://github.com/phpspec/phpspec/blob/b6ef0b627f212910c6e3dfd5b9ad2630b778fff4/src/PhpSpec/Util/Filesystem.php#L55-L61) class already leverages Symfony's Finder to find for spec files in a path, which return instances of `Symfony\Component\Finder\SplFileInfo`, I figured it wouldn't affect the testeability of this class to rely on Symfony's own implementation of `SplFileInfo`.